integration_tests/src/main/python/parquet_testing_test.py

```diff
@@ -72,6 +73,43 @@
     _error_files["lz4_raw_compressed.parquet"] = "Exception"
     _error_files["lz4_raw_compressed_larger.parquet"] = "Exception"
 
+def hdfs_glob(path, pattern):
+    """
+    Finds hdfs files by checking the input path with glob pattern
+
+    :param path: hdfs path to check 
+    :type path: pathlib.Path 
+    :return: generator of matched files
+    """
+    path_str = path.as_posix()
+    full_pattern = path_str + '/' + pattern
+    cmd = ['hadoop', 'fs', '-ls', '-C', full_pattern]
+
+    process = subprocess.Popen(cmd, stdout=subprocess.PIPE, stderr=subprocess.PIPE, universal_newlines=True)
+    stdout, stderr = process.communicate()
+    if process.returncode != 0:
+        raise AssertionError('Failed to list files from hdfs: {}. Error: {}'.format(path_str, stderr))
```

Comment:
Nit: `path_str` already starts with hdfs: so this message is a bit redundant.
```suggestion
        raise AssertionError(f'Failed to list files from {path_str}. Error: {stderr}')
```

Reply:
build

Reply:
build